### PR TITLE
update PSMN configuration to keep up with cluster changes

### PR DESCRIPTION
**Pull request by l-modolo**
---
name: psmn
about: update PSMN configuration to keep up with cluster changes
---

### CODE REVIEW

**Review by jfy133 — APPROVED** [review]

File: conf/psmn.config [hunk 1]
```diff
@@ -13,49 +13,44 @@ charliecloud {
 }
 
 process {
-    resourceLimits = [
-        memory: 370.GB,
-        cpus: 32,
-        time: 96.h
-    ]
```
Comment (pontus) [inline conf/psmn.config:20]: Was this removal intended? Seems like you'd want to keep `resourceLimits` around.

Comment (pontus) [reply conf/psmn.config:20]: Using `clusterOptions` to select partition is not wrong, but one could consider `queue` instead to manage those needing to override `clusterOptions`.

**Review by pontus — APPROVED** [review]

File: conf/psmn.config (hunk 1 above)
Comment (jfy133) [reply conf/psmn.config:20]: Oops, I missed that - yes the resource limits definitely should be included!

Comment (l-modolo) [reply conf/psmn.config:20]: Thank you for the comment, so I can replace :
```
clusterOptions = "--partition=E5,Lake"
```

by :

```
queue = "E5,Lake"
```

with slurm ?

Comment (pontus) [reply conf/psmn.config:20]: Yes.

**Comment by l-modolo** [comment]
Hello,

I applied all the recommended modifications, please tell me if there is anything else I should modify.

Best,

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
You're good to merge @l-modolo !

**Comment by l-modolo** [comment]
Hi,

Maybe I missed something, but how do I merge this pull request, now that it has been approved ?
I thought that it was already merged, but apparently it's not the case.

Best,
Laurent

**Comment by l-modolo** [comment]
Hi @jfy133,

sorry to bother you again, but how do I merge this pull request to the master branch ?

Best,
Laurent

**Comment by jfy133** [comment]
Do you not have a merge pull request button?

image

Are you already in the github nfcore organisation?? https://nf-co.re/join

Otherwise I'll merge on your behalf

**Comment by l-modolo** [comment]
No I don't think I had one, I could only update my depos to match the last commit from this one.

Anyway, thank you for the merge !
